src/view/ClusterDetailsPanel.ts

```diff
@@ -181,17 +181,27 @@ export class ClusterDetailsPanel {
 						// get managed cluster
 						clusterManager = await this._kubeDataLoader.getResources(selectedCluster, "ClusterManager");
 						this._panel.webview.postMessage({"clusterManager":JSON.stringify(clusterManager)});
-					} else {
+					}
+					try {
+						klusterlet = await this._kubeDataLoader.getResources(selectedCluster, "Klusterlet");
+
+					} catch (exceptionVar) {
+						klusterlet = [];
```

Comment:
for this version write exception to log

Reply:
try/catch removed and replaced by suitable return statement in kube.ts